optuna/visualization/_pareto_front.py

```diff
@@ -121,6 +121,47 @@ def objective(trial):
 
     Returns:
         A :class:`plotly.graph_objs.Figure` object.
+
+
+    Example:
```

Comment:
Please move this example above the `Args` section.

Reply:
resolved in bba7188527cf2e21b5992115956c9f22834a8860